pyproject.toml

```diff
@@ -38,6 +38,7 @@ numpy = "2.2.3"
 scipy = "1.15.2"
 scs = "3.2.7.post2" 
 picos = "2.6.0"
+jupyter-sphinx = "^0.5.3"
```

Comment:
Didn't you already add `jupyter-sphinx` as a dependency in a separate PR? It might be better to rebase your branch then.